app/javascript/services/fetchMappingTerms.jsx

```diff
@@ -0,0 +1,16 @@
+import apiService from "./apiService";
+
+const fetchMappingsTerms = async (mappingId) => {
+  const response = await apiService.get(
```

Comment:
If a request isn't successful, this method will throw an exception instead of returning a response with a non-200 status code. So those exceptions need to be handled (here and elsewhere).

Reply:
@excelsior I implemented a new layer to handle errors and more and applied only to this request. If you think that's ok I can continue to apply it to the rest of the services connector components.

Reply:
Nicely done! Let's expand this approach to cover the rest of the API calls.

Reply:
@excelsior I'll merge it and correct the change of the files in a different PR, because it's going to get big.